### DISCUSSION TITLE
Error en test hard en order?

### DISCUSSION THREAD

**Discussion by MaximilianWilke** [IIC2133-PUC/2024-2#112] [Tarea 1] [answered]
Hola creo que en el test hard de order hay un error en el output esperado ya que en medio de la impresión de los id's de uno de los double-sort-by, ocurre un salto de linea en medio de la impresión, lo que arroja un error al hacer diff. La linea sería la 1385 y 1386 (del output) (Por lo menos cuando abro el archivo en vscode aparece con un salto de linea repentino que corta la cadena de los id

Es la unica linea que me falla

**Comment by sebacarrascop** [comment, marked as answer]
Hola!,
Que extraño, a mi me sale bien.
Podrías volver a descargarlo y ver si continua el problema?
